### PR TITLE
Lps 108860 | Traffic sources

### PR DESCRIPTION
**Pull request by andresfulla**
Not sending a propper PR because the copy is not yet the one. But, could you take a look, please?

### CODE REVIEW

**Comment by andresfulla** [comment]
@dgarciasarai this is ready to review/send. Feedback is welcomed

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js [hunk 1]
```diff
@@ -88,14 +94,27 @@ export default function({context, props}) {
...
 				defaultTimeSpanOption={defaultTimeSpanKey}
 				languageTag={languageTag}
 				timeSpanOptions={timeSpans}
 			/>
+
+			<h5 className="mt-4 sheet-subtitle text-secondary">
+				{Liferay.Language.get('traffic-sources')}
+				<Hint
+					message={Liferay.Language.get(
+						'traffic-sources-help-message'
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js]: This key does not exist in Language.properties

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Hint.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		<>
+			<span
+				className="p-1"
+				onMouseEnter={handleMouseEnter}
+				onMouseLeave={handleMouseLeave}
+				ref={iconRef}
+			>
+				<ClayIcon
+					className="mr-1"
+					small="true"
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Hint.js]: Just `small` does not work here?

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+									</td>
+								</tr>
+							);
+						})}
+					</tbody>
+				</table>
+			</div>
+
+			<div className="pie-chart-wrapper--chart">
+				<PieChart height={80} width={80}>
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/TrafficSources.js]: What do you think about extract this hardcoded values to a constant?

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -22,4 +27,6 @@ this-number-refers-to-the-total-number-of-reads-since-the-content-was-published=
 this-number-refers-to-the-total-number-of-views-since-the-content-was-published=This number refers to the total number of views since the content was published.
 total-reads=Total Reads
 total-views=Total Views
+traffic-sources=Traffic Sources
+traffic-sources-message=Sources that drive people to your page. They're calculated on average per month.
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: Other messages are `*-help-message`. Also, the key used in the UI is `traffic-sources-help-message`.

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss [hunk 5]
```diff
@@ -113,4 +113,24 @@
 	.interval-selector-wrapper {
 		display: flex;
 	}
+
+	.pie-chart-wrapper {
+		display: flex;
+		justify-content: space-between;
+		.pie-chart-wrapper--legend--title {
+			font-weight: 600;
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss]: Total views, total reads, and legend don't have this `font-weight` attribute. Double-check with Susana ✔️✔️😂

**Comment by dgarciasarai** [comment]
I put you some comments, but in general, it's awesome! 🚀You rock! 😍

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js [hunk 6]
```diff
@@ -68,8 +70,12 @@ export default function({context, props}) {
 				title={title}
 			/>
 
+			<h5 className="mt-4 sheet-subtitle text-secondary">
+				{Liferay.Language.get('reads-and-views')}
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/AnalyticsReportsApp.js]: In the figma file, this message is `VIEWS AND READS`

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss (hunk 5 above)
Comment (andresfulla) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss]: These are meant to work as links very soon. So I'll fix them there.

**Comment by andresfulla** [comment]
@dgarciasarai thanks!! triple nice

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:stable

**Comment by andresfulla** [comment]
@dgarciasarai I just pushed a fix for the `recharts` module. That should work now 🤞

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by dgarciasarai** [comment]
Pull request submitted to https://github.com/dgarciasarai/liferay-portal/pull/182. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by dgarciasarai** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/86205. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by dgarciasarai** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/86212. [See changes here.](https://github.com/dgarciasarai/liferay-portal/compare/c8c094a4ee25db~2...c8c094a4ee25db) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**